.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,70 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Feature Request
+description: Request new or improved functionality or changes to existing cuDF functionality
+title: "[FEA]: "
+labels: ["feature request"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to help cuDF and fill out this feature request!
+        - By submitting this issue, you agree to follow our [Code of Conduct](https://docs.rapids.ai/resources/conduct/)
+        - You also confirm that you have searched the [open feature requests](https://github.com/rapidsai/cuDF/issues?q=is%3Aopen+is%3Aissue+label%3A%22feature+request%22%2Cimprovement%2Cenhancement) and have found no duplicates for this request
+
+  - type: dropdown
+    id: new_or_improvement
+    attributes:
+      label: Is this a new feature, an improvement, or a change to existing functionality?
+      options:
+        - New Feature
+        - Improvement
+        - Change
```

Comment:
We don't currently track the differences here -- we have to label all of these things as `improvement` for the label checker. I think the difference is often fuzzy and thus a potential source of friction for the person filing an issue.

Reply:
The label checker [does differentiate](https://docs.rapids.ai/resources/label-checker/) between `feature request` and `improvement` which to me indicates the difference between `New` vs `Improvement`, but agree it's not captured in `Change`.

Intention here is to help the team recognize if a user is asking for a change to how things work vs just an addition. It's a focusing tool, but if it doesn't help, as with the other things, happy to remove!

Reply:
TIL that the "feature request" label exists for PRs and not just issues. 👍 Thus far I have always marked "improvement."

I'm totally happy to keep or drop most of these questions, I just want to minimize the form length and maximize the value added!